cmd/flags.go

```diff
@@ -249,6 +250,12 @@ var (
 		Usage: "Specifies the size in bytes of bolt db's mmap syscall allocation",
 		Value: 536870912, // 512 Mb as a default value.
 	}
+	// ApiTimeoutFlag specifies the timeout value for API requests in minutes. A timeout of zero means no timeout.
+	ApiTimeoutFlag = &cli.IntFlag{
+		Name:  "api-timeout",
+		Usage: "Specifies the timeout value for API requests in minutes",
+		Value: int(2 * time.Minute),
```

Comment:
You are asking the user to put time in minutes, but here you have the time in nanoseconds.

Reply:
I think reverting 5da5489 would bring back minutes as it was the only change that commit brought. Still I wonder if it's not a valid usecase to have timeouts lower than a minute